src/Util/PaymentRequestUtil.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Class PaymentRequestUtil
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Util;
+
+/**
+ * Util class for loading, sanitizing, and escaping data from payment requests.
+ */
+class PaymentRequestUtil {
+
+	/**
+	 * Get the fraud prevention token from the request.
+	 *
+	 * @return string|null
+	 */
+	public static function get_fraud_prevention_token(): ?string {
```

Comment:
Context: a comment from [this doc](https://github.com/Automattic/woocommerce-payments/tree/develop/src#legacy-proxy):

>  Use this for non-`src` classes. `src` classes should only have pure static methods.

Let's not make this class a utility, or keep its methods static. Static methods should be pure, and accessing `$_POST` here is not pure. It might be easy to test this class now, but we'll be making this a bit weirder when testing services, which uses this one because static methods cannot be mocked. We'd be forcing devs to check the static method to understand what `$_POST` parameters to modify.

I also don't like that we need to access/manipulate (for tests) $_POST here. When I think about it, I don't really like [my implementation](https://github.com/Automattic/woocommerce-payments/blob/spike/payment-process-states/includes/payment/class-post-request.php) either. Let's use `$_POST` as a constructor parameter for the class, and then use something like `$this->request['wcpay-fraud-prevention-token']. WDYT?

Reply:
All good points. Going to change the approach now.